src/MenuItem/ToggableTrait.php

```diff
@@ -2,57 +2,20 @@
 
 namespace PhpSchool\CliMenu\MenuItem;
 
-use PhpSchool\CliMenu\MenuStyle;
```

Comment:
Can we drop the trait please. I'd rather have a bit of duplicated code.

Reply:
Was following your example with `SelectableTrait`. Will remove.

Reply:
Yeah that code is quite old now I guess. I rather not introduce more thinking about it :)

Reply:
Just realized this isn't my branch anymore so I can't push changes

Reply:
@jtreminio you can just send a PR to this branch